net/i2pd/Makefile

```diff
@@ -36,45 +35,34 @@ define Package/i2pd
 endef
 
 define Package/i2pd/description
-	I2P (Invisible Internet Protocol) is a universal anonymous network layer. All
-	communications over I2P are anonymous and end-to-end encrypted, participants
-	don't reveal their real IP addresses.
+  I2P (Invisible Internet Protocol) is a universal anonymous network layer. All
+  communications over I2P are anonymous and end-to-end encrypted, participants
+  don't reveal their real IP addresses.
 endef
 
+TARGET_LDFLAGS+=-latomic
+
 define Package/i2pd/conffiles
 /etc/config/i2pd
 /etc/i2pd/i2pd.conf
 /etc/i2pd/tunnels.conf
-/etc/i2pd/tunnels.d/
-endef
-
-define Build/Prepare
-	$(Build/Prepare/Default)
-	# race condition
-	cd $(PKG_BUILD_DIR); $(MAKE) mk_obj_dir
+/etc/i2pd/tunnels.d/*
 endef
 
-TARGET_LDFLAGS+=-latomic
-
 define Package/i2pd/install
 	$(INSTALL_DIR) $(1)/usr/sbin
 	$(INSTALL_BIN) $(PKG_BUILD_DIR)/i2pd $(1)/usr/sbin
+	$(INSTALL_DIR) $(1)/etc/init.d
+	$(INSTALL_BIN) ./files/i2pd.init $(1)/etc/init.d/i2pd
 	$(INSTALL_DIR) $(1)/usr/share/i2pd
 	$(CP) $(PKG_BUILD_DIR)/contrib/certificates  $(1)/usr/share/i2pd
 	$(INSTALL_DIR) $(1)/etc/config
-	$(INSTALL_DATA) ./files/i2pd.config $(1)/etc/config/i2pd
+	$(INSTALL_CONF) ./files/i2pd.config $(1)/etc/config/i2pd
 	$(INSTALL_DIR) $(1)/etc/i2pd
-	$(INSTALL_CONF) $(PKG_BUILD_DIR)/contrib/i2pd.conf $(1)/etc/i2pd
-	$(SED) ' \
-		s/127.0.0.1/192.168.1.1/g; \
-		s/datadir = \/var\/lib/datadir = \/etc/ \
-	' $(1)/etc/i2pd/i2pd.conf
-	$(INSTALL_CONF) $(PKG_BUILD_DIR)/contrib/tunnels.conf $(1)/etc/i2pd
-	$(INSTALL_DIR) $(1)/etc/i2pd/tunnels.d
-	$(INSTALL_CONF) $(PKG_BUILD_DIR)/contrib/tunnels.d/*.conf  $(1)/etc/i2pd/tunnels.d
+	$(INSTALL_DATA) $(PKG_BUILD_DIR)/contrib/i2pd.conf $(1)/etc/i2pd
 	# subscriptions.txt deprecated, see #8088
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/i2pd.init $(1)/etc/init.d/i2pd
+	$(INSTALL_DIR) $(1)/etc/i2pd/tunnels.d
```

Comment:
This seems to be empty folder, right? It will be deleted. There should be atleast some empty file. What I understand is that there will be some files once i2pd is started as it creates symlink, right?

Reply:
I can make copy of README file about that directory.